### DISCUSSION TITLE
Why are the functions in `mitiq/interface/conversions.py` missing from the API-doc?

### DISCUSSION THREAD

**Discussion by purva-thakre** [unitaryfund/mitiq#2408] [Q&A] [answered]
I have been working on understanding how `accept_qprogram_and_validate` works. This made me realize the functions in `mitiq/interface/conversions.py` are missing from the `Interface` section of the API-doc.

Some of the functions used by `accept_qprogram_and_validate` are also missing their docstrings. This makes it difficult to understand what the goal of a particular function is.

**Comment by purva-thakre** [comment]
Plus, I think the docs section on [`Frontends and Backends`](https://mitiq.readthedocs.io/en/stable/guide/frontends-backends.html#frontends-and-backends) needs to be updated with information about unsupported circuit types.

(if I am reading the docstring of the following function correctly: a user can define their own circuit converter)

https://github.com/unitaryfund/mitiq/blob/bf82e84410e1805437e15f1d26b0c5be8e8dcb8a/mitiq/interface/conversions.py#L37-L49

**Reply by natestemen** [reply]
Good point, I don't think we've added public facing documentation for this feature! Care to open an issue?

**Comment by natestemen** [comment, marked as answer]
My rationalization for the status quo is that the API-doc is primarily for Mitiq users, and Mitiq users should basically never user any of the functions in `mitiq/interface/conversions.py`.

**Reply by purva-thakre** [reply]
well, if we create an issue for the `register_mitiq_converters` function, we should also add all the functions used by the converter. WDYT?

**Reply by jordandsullivan** [reply]
Plan to test out which functions are needed by an end user who wants to register a converter in this way and add those to API doc.

**Comment by purva-thakre** [comment]
Resolved in favor of https://github.com/unitaryfund/mitiq/issues/2409
